### DISCUSSION TITLE
PIXI.RenderTexture.clone() return a same texture?

### DISCUSSION THREAD

**Discussion by znw-test** [pixijs/pixijs#8849] [Q&A]
I use PixiJS v6.2.2. I tried to clone each of my renderings and switch back and forth between them.
```js
list.push(renderTexture1.clone())
// ......
sprite.texture = list[index]
```
I tried to change the Sprite texture directly to a clone texture, but I found it didn't seem to work.
I'm going to have to do it like this:
```js
let cloneResult = new RenderTexture.create({ width, height })
app.renderer.render(quadFinal, { renderTexture: cloneResult })
list.push(cloneResult)
// ......
sprite.texture = list[index]
```
But I find this method error-prone when used multiple times.

**Comment by bigtimebuddy** [comment]
Small thing: `new RenderTexture.create` should be `RenderTexture.create` (no `new`)

**Reply by znw-test** [reply]
Thank you for your answer. 

Yes, I have corrected the problem. May I ask is this method a good one?
```js
let cloneResult = RenderTexture.create({ width, height })
app.renderer.render(quadFinal, { renderTexture: cloneResult })
list.push(cloneResult)
// ......
sprite.texture = list[index]
```
